### ISSUE TITLE
Minifying Strings As Short As Possible

### ISSUE THREAD

**Issue by hadiidbouk** [rockbruno/swiftshield#29]
Hello,

It will be better I think to add an option that minifying the obfuscated strings as short as possible.

**Comment by rockbruno** [comment]
Do you mean `-obfuscation-character-count`? It controls the size of the names.

**Comment by hadiidbouk** [comment]
Yes, but this option is manual, it will be better to obfuscate as short as possible, if a class has a lot of properties/methods, maybe 1 character of obfuscation isn't enough.

What do you think?

**Comment by rockbruno** [comment]
Hmmm it kinda goes against the point of obfuscation, don't you think? You want things to be as confusing as possible so attackers have no idea what's going on. It would be pretty simple to figure out what a class is doing if everything is just one character. Is your concern the binary size?

**Comment by hadiidbouk** [comment]
You are right, If we programmatically specify the count of obfuscation we give the attacker an information for free.

Yes, but will pick a 3-4 characters for my libraries
